server/app.js

```diff
@@ -235,49 +241,57 @@ app.get('/api/related', (req, res) => {
       const distances = sources.map(source => {
         const keys = Object.keys(source)
 
-        const more_like_this_keys = MORE_LIKE_THIS_FIELDS.map(field => {
-          return (field[field.length-1] === '*') ? field.slice(field, field.length-1) : field
+        const moreLikeThisKeys = MORE_LIKE_THIS_FIELDS.map(field => {
+          return (field[field.length - 1] === '*') ? field.slice(field, field.length - 1) : field
         })
 
-        const filtered_keys = keys.filter(key => more_like_this_keys.some(more_like_this_key => key.indexOf(more_like_this_key) === 0 ))
+        const filteredKeys = keys.filter(key => moreLikeThisKeys.some(moreLikeThisKey => key.indexOf(moreLikeThisKey) === 0))
 
-        const descriptor_keys = filtered_keys.filter(key => key in objectDescriptors)
+        const descriptorKeys = filteredKeys.filter(key => (key !== 'people') && key in objectDescriptors)
 
-        const distance = descriptor_keys.reduce((sum, key) => {
-          const absolute_distance = parseFloat(objectDescriptors[key]) - parseFloat(source[key])
-          return sum + (absolute_distance * absolute_distance)
+        const distance = descriptorKeys.reduce((sum, key) => {
+          const absoluteDistance = parseFloat(objectDescriptors[key]) - parseFloat(source[key])
+          return sum + (absoluteDistance * absoluteDistance)
         }, 0)
 
-        const normalized_distance =  distance / descriptor_keys.length
+        let normalizedDistance = distance / descriptorKeys.length
+
+        if (source.people !== objectDescriptors.people) {
+          normalizedDistance += 250
+        }
 
-        return normalized_distance
+        return normalizedDistance
       })
 
-      const sortedDistances = distances.slice().sort();
+      const sortedDistances = distances.slice().sort()
 
-      const median = sortedDistances[Math.floor(distances.length/2)]
+      const quartiles = [
+        sortedDistances[Math.floor(1 * distances.length / 4)],
+        sortedDistances[Math.floor(2 * distances.length / 4)],
+        sortedDistances[Math.floor(3 * distances.length / 4)]
+      ]
```

Comment:
switched to quartiles, so we consider the bottom 25% and top 25% to increase diversity